lib/components/table.vue

```diff
@@ -17,7 +17,8 @@
                     :tabindex="field.sortable?'0':null"
                 >
                   <slot :name="'HEAD_'+key" :label="field.label" :column="key" :field="field">
-                    <div v-html="field.label"></div>
```

Comment:
This line will be rendered if the user does not provide content for the named slot.

If the user does provide a scoped slot, then its contents (HTML, text, components, etc) will replace this default.